README.md

```diff
@@ -1,4 +1,4 @@
-<img src="http://www.mit.edu/~kirschen/img/GPfit_logo.png" width=110 alt="GPkit" />
+<img src="https://github.com/hoburg/gpfit/raw/master/docs/source/GPfit_logo.png" width=110 alt="GPfit logo" />
```

Comment:
note this is slightly different from how we do it in GPkit (`[<img src="http://gpkit.rtfd.org/en/latest/_images/gplogo.png" width=110 alt="GPkit" />](http://gpkit.readthedocs.org/)`). Not sure which is better but maybe we should be consistent.